hack/sss-hcp.yaml

```diff
@@ -0,0 +1,101 @@
+  value: quay.io/app-sre/managed-release-bundle-hcp
+metadata:
+  name: selectorsyncset-template
+objects:
+  - apiVersion: hive.openshift.io/v1
+    kind: SelectorSyncSet
+    metadata:
+      name: rvmo-hcp
+    spec:
+      clusterDeploymentSelector:
+        matchLabels:
+          api.openshift.com/managed: 'true'
+        matchExpressions:
+        - key: ext-hypershift.openshift.io/cluster-type
+          operator: In
+          values:
+          - service-cluster
+          - management-cluster-v2
+      resourceApplyMode: Sync
+      resources:
+        - apiVersion: apps.open-cluster-management.io/v1
+          kind: PlacementRule
+          metadata:
+            name: rvmo-hcp
+            namespace: openshift-acm-policies
+          spec:
+            clusterSelector:
+              matchExpressions:
+              - key: hypershift.open-cluster-management.io/hosted-cluster
+                operator: In
+                values:
+                - 'true'
+              - key: openshiftVersion-major-minor
+                operator: In
+                values:
+                - "4.16"
+        - apiVersion: policy.open-cluster-management.io/v1
+          kind: PlacementBinding
+          metadata:
+            name: rvmo-hcp
+            namespace: openshift-acm-policies
+          placementRef:
+            name: rvmo-hcp
+            kind: PlacementRule
+            apiGroup: apps.open-cluster-management.io
+          subjects:
+            - name: rvmo-hcp
+              kind: Policy
+              apiGroup: policy.open-cluster-management.io
+        - apiVersion: policy.open-cluster-management.io/v1
+          kind: Policy
+          metadata:
+            name: rvmo-hcp
+            namespace: openshift-acm-policies
+          spec:
+            remediationAction: enforce
+            disabled: false
+            policy-templates:
+              - objectDefinition:
+                  apiVersion: policy.open-cluster-management.io/v1
+                  kind: ConfigurationPolicy
+                  metadata:
+                    name: rvmo-hcp
+                  spec:
+                    pruneObjectBehavior: DeleteIfCreated
+                    object-templates:
+                      - apiVersion: v1
+                        kind: ConfigMap
+                        metadata:
+                          name: osd-cluster-metadata
+                          namespace: openshift-config
+                        data:
+                          foo: bar
+                      - apiVersion: package-operator.run/v1alpha1
+                        kind: ClusterObjectTemplate
+                        metadata:
+                          name: managed-openshift-release-bundle-template
+                        spec:
+                          sources:
+                            - apiVersion: v1
+                              items:
+                                - destination: .
+                                  key: .data
```

Comment:
there's no ClusterDeployment to get data from, also we don't use ManagedCluster labels for conditional rollouts, so at this point I'm not even sure if we need a ConfigMap at all in HCP.